src/main/java/ua/mateacademy/tania/lesson17/jdbc/CRUDTemplateAbstract.java

```diff
@@ -0,0 +1,25 @@
+package ua.mateacademy.tania.lesson17.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public abstract class CRUDTemplateAbstract<T> {
+    public boolean templateOperation(T t) throws SQLException {
+        Connection connection = null;
+        PreparedStatement preparedStatement = null;
+        Boolean isDone;
```

Comment:
Why don't you initialize variable isDone?

Reply:
> Why don't you initialize variable isDone?

because of boolean type in java has default value false